kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js

```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
             temperature: 800,
             time: 58,
             id: 'tconstruct:smeltery/melting/metal/iron/chain'
+        },
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
+            time: 58,
+            id: '${id_prefix}molten_gold/chain'

```

Comment:
This id is different from the rest, why is that?

Reply:
I am actually not sure about it.